oneclick_monolith_install/install-config.yml

```diff
@@ -0,0 +1,18 @@
+# Name to uniquely identify your CommCareHQ instance/environment.
+env_name: ""
+
+# Web address of your HQ instance
```

Comment:
```suggestion
# Web address of your HQ instance. e.g. "example.com"
```

Reply:
:) I had exactly the same! I removed it by mistake.